kythe/go/extractors/gcp/README.md

```diff
@@ -45,22 +45,20 @@ _REPO_NAME=https://github.com/project-name/repo-name\
 --no-source
 ```
 
-This Cloud Build uses two artifacts in gcr.io/kythe-public:
+### Guava specific example
 
-### Extractor Artifacts
+To extract multiple parts of https://github.com/google/guava, use
+`examples/guava-mvn.yaml`.
 
-gcr.io/kythe-public/kythe-javac-extractor-artifacts created from
-`kythe/java/com/google/devtools/kythe/extractors/java/artifacts` contains:
-
-* `javac-wrapper.sh` script which calls Kythe extraction and then an actual java
-  compiler
-* `javac_extractor.jar` which is the Kythe java extractor
-* `javac9_tools.jar` which contains javac langtools for JDK 9, but targets JRE 8
+```
+gcloud builds submit --config examples/mvn.yaml \
```

Comment:
Is the config supposed to be "examples/guava-mvn.yaml" like you stated above or are they supposed to be different files?